oap-server/server-library/library-client/src/test/java/org/apache/skywalking/oap/server/library/client/elasticsearch/ITElasticSearchClient.java

```diff
@@ -183,4 +187,51 @@ public void bulk() throws InterruptedException {
         bulkProcessor.flush();
         bulkProcessor.awaitClose(2, TimeUnit.SECONDS);
     }
+
+    @Test
+    public void timeSeriesOperate() throws IOException {
+        String indexName = "test_time_series_operate";
+        String timeSeriesIndexName = indexName + "-2019";
+
+        try {
+            if (client.isExistsTemplate(indexName)) {
```

Comment:
Why these is an chance existing? If other case doesn't remove, set a finally there.

Reply:
Because my local tests don't create a new es container every time, I delete it